### ISSUE TITLE
Make WordVault tiles a bit more readable

### ISSUE THREAD

**Issue by domino14** [domino14/Webolith#481]
![image](https://github.com/user-attachments/assets/99382abf-a971-45d6-9f2a-6583a0bda3e3)

![image](https://github.com/user-attachments/assets/5ec45a88-a905-4a4b-b1c8-242258a7c44f)

Here are some screenshots of WordVault and WordWalls on a phone.
Maybe it's a combination of the letter size / tile spacing / contrast but it seems to me that the WordVault card is harder to read. Could we experiment with some of these parameters a bit? 

The WordWalls tiles are purposely oversized (letter to tile ratio), we probably don't need to go that high to make WordVault more readable. Let me know what you think. @BenMusch

**Comment by BenMusch** [comment]
@domino14 Yeah I agree here -- studied a bit on mobile while walking the dog and I think a higher front-to-tile ratio is best. Not all that different from the scrabble tiles designed for people with impaired vision
![image](https://github.com/user-attachments/assets/f8539c6f-42ea-4b7b-96e0-9a8d2da0490e)

**Comment by domino14** [comment]
Check this out https://github.com/domino14/Webolith/pull/485 -- we can iterate on this later if you want. I think making those  letters bigger on mobile fixed a lot of the problem.
